src/main/java/frc/robot/commands/climber/Climb.java

```diff
@@ -36,14 +34,13 @@ public void initialize() {
 
   @Override
   public void execute() {
-    m_climber.set(Math.sqrt(Math.pow(m_stick.getRawAxis(m_axis1), 2) + Math.pow(m_stick.getRawAxis(m_axis2), 2))
-        * Constants.CLIMBER_MAX_PERCENT_OUTPUT);
+    m_climber.set(m_speed.getAsDouble());
   }
 
   @Override
   public void end(boolean interrupted) {
     m_climber.stop();
-    m_climber.setPiston(Value.kOff);
+    m_climber.setPiston(interrupted ? Value.kReverse : Value.kOff);
```

Comment:
Can you add a comment here explaining the logic you explained to me?

Reply:
Resolved in bbd7d1a